CHANGELOG.txt

```diff
@@ -1,12 +1,16 @@
+Version 1.4.0 2020-10-26
```

Comment:
We usually use the [`gem-release`](https://github.com/apartmentlist/scripts/blob/main/bin/gem-release) for tagging, bumping gem version, and updating the changelog. So after your changes get merged to `main`, you can run this script like, 
```bash
path/to/gem-release [major/minor/patch] "Short message with changes"
```

I would remove these manually changes and use the script.